test/e2e/probes_test.go

```diff
@@ -0,0 +1,151 @@
+// +build e2e
+
+/*
+Copyright 2020 Kohl's Department Stores, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	util "github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
+	"io/ioutil"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestReadinessAndLivelinessProbes(t *testing.T) {
+	ctx := framework.NewTestCtx(t)
+	defer ctx.Cleanup()
+
+	operatorName, found := os.LookupEnv("OPERATOR_NAME")
+	if !found {
+		operatorName = "eunomia-operator"
+	}
+	operatorNamespace, found := os.LookupEnv("OPERATOR_NAMESPACE")
```

Comment:
I think the namespace should be retrieved using the method `ctx.GetNamespace()`.

Ref: https://github.com/KohlsTechnology/eunomia/commit/a41add51152126ad0e82a6e84f2b08b289890fa7

Reply:
Hmmm ... no. This `namespace, err := ctx.GetNamespace()` gets temporary namespace prepared for test, but in this test, I'm testing availability of endpoints of operator itself, so I need its namespace, not temporary one.

Reply:
Ahhhh yes, misunderstanding on my part. Thanks!